src/factorizations/factorizations.jl

```diff
@@ -84,15 +81,19 @@ function LinearAlgebra.isposdef!(t::AbstractTensorMap)
 end
 
 # TODO: tolerances are per-block, not global or weighted - does that matter?
-function MatrixAlgebraKit.is_left_isometry(t::AbstractTensorMap; kwargs...)
+function MAK.is_left_isometric(t::AbstractTensorMap; kwargs...)
     domain(t) ≾ codomain(t) || return false
-    f((c, b)) = MatrixAlgebraKit.is_left_isometry(b; kwargs...)
+    f((c, b)) = MAK.is_left_isometric(b; kwargs...)
     return all(f, blocks(t))
 end
-function MatrixAlgebraKit.is_right_isometry(t::AbstractTensorMap; kwargs...)
+function MAK.is_right_isometric(t::AbstractTensorMap; kwargs...)
     domain(t) ≿ codomain(t) || return false
-    f((c, b)) = MatrixAlgebraKit.is_right_isometry(b; kwargs...)
+    f((c, b)) = MAK.is_right_isometric(b; kwargs...)
     return all(f, blocks(t))
 end
+function MAK.isisometric(t::AbstractTensorMap, (p₁, p₂)::Index2Tuple; kwargs...)
```

Comment:
Do we still support a permutation argument here? Didn't we remove this in most other places?